pkgs/development/python-modules/supabase_auth/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, pkgs
+, python3Packages
+, fetchPypi
+, poetry-core
+}:
+
+python3Packages.buildPythonPackage rec {
+  pname = "gotrue";
```

Comment:
?

Reply:
@risicle it's had a recent name change, but I guess we'd want to use the name as found pypi, so I'll revise. Thanks!